cves/kernel/CVE-2020-16166.yml

```diff
@@ -397,9 +336,11 @@ discussion:
 
     Put any links to disagreements you found in the notes section, or any other
     comment you want to make.
-  discussed_as_security:
-  any_discussion:
-  note:
+  discussed_as_security: false
+  any_discussion: false
+  note: |
+    The had no real disagreements. It took 2 attempts to fix but otherwise
```

Comment:
Should be "they" instead of "the". Just a tiny typo

Reply:
Fixed the typo, will be updated in next commit.